### ISSUE TITLE
[Python] Implement new Concept Exercise: floats-basic

### ISSUE THREAD

**Issue by aldraco** [exercism/v3#1026]
This issue describes how to implement the `floats-basic` concept exercise for the Python track.

## Goal

The goal of this exercise is to teach the basics of the `float` data type in Python. The student will have already completed the `int-basic` exercise, so this one can build on that.

## Learning objectives

 - create a float literal
 - create a float by casting an `int`
 - perform integer division and "regular" division and see how they differ
 - round a number using `math.floor()` and `math.ceiling()`
 - changing precision of a float, using `round(<float>, <digits>)`

## Out of scope

 - special values for `Infinity` and `NaN`
 - issues with precision limits
 - Checking strings to see if they are floats. This means any test input to this exercise should not require a student to (a) catch or raise an Exception if a non-float string is passed, or (b) check to see if the string is a float. Only pass valid float strings, i.e. `"1.032"`.

## Concepts

 - `float-basic`

## Prerequisites

 - [`builtin_types`](https://github.com/exercism/v3/tree/master/languages/python/reference/concepts/builtin_types)
 - [`data_types`](https://github.com/exercism/v3/blob/master/languages/python/reference/concepts/data_structures.md)
 - `int-basic`

## Resources to refer to

 - [Python numeric type documentation](https://docs.python.org/3/library/stdtypes.html#typesnumeric)
 - [Documentation for `float()` built in](https://docs.python.org/3/library/functions.html#float)
 - [Documentation for builtin function `round()`](https://docs.python.org/3/library/functions.html#round)

### Hints

Hints can link to the builtin function docs mentioned above.

### After

After the exercise, string formatting can be introduced using float numbers as an example. The [`format()` function](https://docs.python.org/3/library/functions.html#format) and [format spec](https://docs.python.org/3/library/string.html#formatspec) can be shown, with specific emphasis on the examples related to float numbers. Here is a [tutorial](http://anh.cs.luc.edu/python/hands-on/3.1/handsonHtml/float.html) to this effect.

## Representer

TBD.

## Analyzer

TBD.

## Implementing

Tests should be written using `unittest.TestCase`, and the test file named `float_basic_test.py`.

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

**Comment by BethanyG** [comment]
Propose we close this and #1025 in favor of a `numbers basics` exercise where we cover the basic & built in numeric types `float`, `int`, and `complex`.  Then follow up with a numbers advanced that goes over things like `decimal`, `fraction`, and other considerations/methods.  @cmccandless @DavidGerva -- thoughts?

**Comment by cmccandless** [comment]
Sounds good to me!

**Comment by DavidGerva** [comment]
I totally agree! I don't think you can exercise only a simple basic concept like float without some others.
